### PR TITLE
Extension to sort filter values

### PR DESCRIPTION
**Pull request by alexbridge**
Extension to sort filter values

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Shopgate Connect - Sort filters and filters values
+
+Sorting filters and filters values for search and category pages.
```
Comment (Carsten89) [inline README.md]:
```suggestion
Sorting filters and filter values for search and category pages.
```

File: extension-config.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate-project/sort-filters",
+  "configuration": {
+    "sortFiltersByLabel": {
+      "type": "admin",
+      "destination": "backend",
+      "default": false,
+      "params": {
+        "type": "json" ,
```
Comment (Carsten89) [inline extension-config.json]:
```suggestion
        "type": "checkbox" ,
```

File: extension-config.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        "label": "Sort filters by label"
+      }
+    },
+    "sortFilters": {
+      "type": "admin",
+      "destination": "backend",
+      "default": {
+        "display_amount": -10,
```
Comment (Carsten89) [inline extension-config.json]: remove defaults

File: extension-config.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        "type": "json" ,
+        "label": "Priorities for sorting filters"
+      }
+    },
+    "sortValuesByLabel": {
+      "type": "admin",
+      "destination": "backend",
+      "default": false,
+      "params": {
+        "type": "json" ,
```
Comment (Carsten89) [inline extension-config.json]:
```suggestion
        "type": "checkbox" ,
```

File: extension/sortValues.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  const {
+    config: {
+      sortValuesByLabel,
+      sortValues
+    }
+  } = context
+
+  if (sortValuesByLabel) {
+    filters.forEach(filter => {
+      if (filter.type !== 'multiselect') {
```
Comment (Carsten89) [inline extension/sortValues.js]: why do we only handle type multiselect here. what about singleselect for example and potentials others?

**Review by Carsten89 — CHANGES_REQUESTED** [review]

File: extension/sortValues.js (hunk 5 above)
Comment (alexbridge) [reply extension/sortValues.js]: The others are `range` . And values can be only in multiselect

Comment (Carsten89) [reply extension/sortValues.js]:
```suggestion
      if (filter.type === 'range') {
```
?

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**